scripts/karma.js

```diff
@@ -10,6 +10,9 @@ const configFile = join(__dirname, '..', process.argv[2]);
 const integrationTests = join(__dirname, '..', 'test', 'integration', 'spec');
 
 function getTestPaths(path) {
+  if (process.env.FILE) {
```

Comment:
@makarandp0 I'm not entirely clear why this is added?

Reply:
We had a similar check earlier in https://github.com/twilio/twilio-video.js/blob/master/karma/makeconf.js#L9 which did not quite work for some files. Moving it at this level makes it works better.